src/set_theory/game/pgame.lean

```diff
@@ -672,6 +685,152 @@ begin
   exact lt_or_equiv_or_gt_or_fuzzy x y
 end
 
+end pgame
+
+-- This can also used for golf `Set.equiv` lemmas, but I'm not sure where to put it.
```

Comment:
This is a great idea, though probably best for a separate PR. I think this fits in logic/relation.lean.